### DISCUSSION TITLE
Didn't approved yet, how long it will take

### DISCUSSION THREAD

**Discussion by chandula00** [community/community#56859] [GitHub Education] [answered]
### Hi everyone,

![image](https://github.com/community/community/assets/115539929/a2702d97-9b6d-4cff-b686-e1e0985cffe9)

I have submitted this several times, but still no response. how long it will take

**Comment by tshenolo** [comment, marked as answer]
Hi @chandula00 

I assume you are requesting approval for a PRO account. I think you have to figure out who within your organization manages your Github account (school administrator). The message in red just means that the school administrator should be able to approve your request.
I recently submitted my request via the same form and it took 4 days for Github to approve it.

I hope that helps
